pilot/utils/llm_connection.py

```diff
@@ -588,3 +599,48 @@ def postprocessing(gpt_response: str, req_type) -> str:
 
 def load_data_to_json(string):
     return json.loads(fix_json(string))
+
+
+
+def stream_anthropic(messages, function_call_message, gpt_data, model_name = "claude-3-sonnet-20240229"):
+    try:
+        import anthropic
```

Comment:
This is imported here so anyone updating (including through the extension) doesn't need to install the anthropic SDK unless they actually use this.